relays/ethereum/src/cli.yml

```diff
@@ -61,10 +62,17 @@ subcommands:
                 value_name: ETH_CONTRACT
                 help: Address of deployed bridge contract.
                 takes_value: true
+                required: true
+            - eth-chain-id:
+                long: eth-chain-id
+                value_name: ETH_CHAIN_ID
+                help: Chain ID to use for signing.
+                required: true
```

Comment:
Making `eth-chain-id` required without making e.g. `eth-signer` required seems strange to me. Let's either mark both `required` or make both work with OE instant seal chain by default. I would prefer to leave everything with default values for now so that we'll be able to work with our Substrate runtime and OE instant-seal network by default. We may revisit cli args in #162 later - this PR makes cli args a bit inconsistent.